cfme/containers/overview.py

```diff
@@ -25,4 +51,5 @@ def step(self):
         self.prerequisite_view.navigation.select('Compute', 'Containers', 'Overview')
 
     def resetter(self):
-        pass
+        # We should wait ~2 seconds for the StatusBox population
+        wait_for(lambda: status_boxs[ContainersProvider].value(), num_sec=10, delay=1)
```

Comment:
My suggestion above was to avoid the above status_boxs variable entirely and be able to do this
```wait_for(lambda: ContainersProvider.PLURAL, num_sec=10, delay=1)```